routers/web/user/auth.go

```diff
@@ -673,6 +666,57 @@ func SignInOAuthCallback(ctx *context.Context) {
 	handleOAuth2SignIn(ctx, loginSource, u, gothUser)
 }
 
+func claimValueToStringSlice(claimValue interface{}) []string {
+	var groups []string
+
+	switch rawGroup := claimValue.(type) {
+	case []string:
+		groups = rawGroup
+	default:
+		str := fmt.Sprintf("%s", rawGroup)
+		if strings.Contains(str, ",") {
+			groups = strings.Split(str, ",")
+		} else {
+			groups = []string{str}
+		}
```

Comment:
We do not need to check ","  

`strings.Split(str, ",")` always does the right thing even if there is no "," in string